core/commands/add.go

```diff
@@ -49,7 +49,7 @@ remains to be implemented.
 	},
 
 	Arguments: []cmds.Argument{
-		cmds.FileArg("path", true, true, "The path to a file to be added to IPFS").EnableRecursive().EnableStdin(),
```

Comment:
oh :+1: on the comment, for sure. fixing the stdin / file distinction seems more trouble than useful atm. sorry for being unclear!